Containerfile.dev

```diff
@@ -0,0 +1,10 @@
+FROM registry.fedoraproject.org/fedora:latest
```

Comment:
This should be :rawhide, makes it easier to find changes in Fedora sooner than at release time :)

Reply:
hmm - I'll think about this, as when using this in the context of "getting started" I rather want it to be working more than fail soon.
Hmm.. I can override this also from "above" this also needs a good documentation about the rationale, thanks!

Reply:
…or can we leave "latest" here as in the `Containerfile.test` there _is_ "rawhide"?

Reply:
Ah, right, for simple getting started latest is better, but an override would be nice.